FIAT/expansions.py

```diff
@@ -52,10 +62,15 @@ def jacobi_factors(x, y, z, dx, dy, dz):
     return fa, fb, fc, dfa, dfb, dfc
 
 
-def dubiner_recurrence(dim, n, order, ref_pts, jacobian):
+def dubiner_recurrence(dim, n, order, ref_pts, Jinv, scale, variant=None):
```

Comment:
A comment perhaps on the arguments and their role?  (Not that it was all that well documented before)